src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js

```diff
@@ -106,6 +106,113 @@ describe('features/content-explorer/content-explorer/ContentExplorer', () => {
             expect(wrapper.find('ContentExplorerSelectAll').prop('isSelectAllChecked')).toEqual(isSelectAllChecked);
         });
 
+        test('should render ContentExplorerIncludeSubfolders with canIncludeSubfolders = true and isSelectAllAllowed = true', () => {
+            const isSelectAllAllowed = true;
+            const canIncludeSubfolders = true;
+            const wrapper = renderComponent({ canIncludeSubfolders, isSelectAllAllowed });
+
+            expect(wrapper.exists('ContentExplorerIncludeSubfolders')).toBeTruthy();
+        });
+
+        test('should not render ContentExplorerIncludeSubfolders with canIncludeSubfolders = false and isSelectAllAllowed = true', () => {
+            const isSelectAllAllowed = true;
+            const canIncludeSubfolders = false;
+            const wrapper = renderComponent({ canIncludeSubfolders, isSelectAllAllowed });
+
+            expect(wrapper.exists('ContentExplorerSelectAll')).toBeTruthy();
+            expect(wrapper.find('ContentExplorerIncludeSubfolders').length).toBe(0);
+        });
+
+        test('should not render ContentExplorerIncludeSubfolders nor ContentExplorerSelectAll with canIncludeSubfolders = true and isSelectAllAllowed = false', () => {
+            const isSelectAllAllowed = false;
+            const canIncludeSubfolders = true;
+            const wrapper = renderComponent({ canIncludeSubfolders, isSelectAllAllowed });
+
+            expect(wrapper.find('ContentExplorerSelectAll').length).toBe(0);
+            expect(wrapper.find('ContentExplorerIncludeSubfolders').length).toBe(0);
```

Comment:
these and the tests below switched from using `exist` to `find`/`length`. we should use one approach